src/assets/css/_sass/elements/_navs.scss

```diff
@@ -3,3 +3,40 @@
   Navs
 
 */
+
+#nav-header {
+  position: absolute;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  padding: 30px 50px;
+  width: 100%;
```

Comment:
`width: 100%` - element is a block which will default to 100% of the parent container.